### ISSUE TITLE
Inline images containing "EI" in image data

### ISSUE THREAD

**Issue by gustavotcabral** [veraPDF/veraPDF-library#1098]
I'm having problems with inline images like the following (note the "EI" in image data):

```
BI
/Width 1
/Height 1
/ColorSpace [ /CalRGB << /Gamma [2.2 2.2 2.2] /Matrix [ 0.4124 0.2126 0.0193 0.3576 0.7152 0.1192 0.1805 0.0722 0.9505 ] /WhitePoint [ 0.950456 1 1.08906 ] >> ]
/BitsPerComponent 8
/Filter [ /A85 ]
ID
EI00 ~>
EI
```
[inline_ei.pdf](https://github.com/veraPDF/veraPDF-library/files/4949292/inline_ei.pdf)

**Actual result:** _6.2.10. A content stream shall not contain any operators not defined in PDF Reference even if such operators are bracketed by the BX/EX compatibility operators_

Is this the intended behavior?

**Comment by THausherr** [comment]
That (EI in inline image data) is a problem with many projects.

Btw the inline image has an illegal colorspace ("It shall not be a CIE-based colour space or a special colour space, with the exception of a limited form of Indexed colour space whose base colour space is a device space and whose colour table is specified by a byte string").

**Comment by bdoubrov** [comment]
Fixed in the latest dev build 1.17.2
